### ISSUE TITLE
Maximum call stack size exceeded in getNeighbouringTimeline

### ISSUE THREAD

**Issue by bkil** [ajbura/cinny#1817]
### Describe the bug

I clicked on something, maybe an unread room in the room drawer. The whole screen went blank and the computer became unresponsive. This is what I found in the console.
```
index-eaa22fd4.js:39 RangeError: Maximum call stack size exceeded
    at e.getNeighbouringTimeline (index-eaa22fd4.js:57)
    at qk (index-eaa22fd4.js:57147)
```

### Reproduction

I don't think I can reproduce it.

### Platform and versions

Cinny v3.2.0 on the web

**Comment by kfiven** [comment]
Can you please fill "Platform and versions" section in bug report?

**Comment by kfiven** [comment]
Closing as,
a). We don't have ways to reproduce 
b). Probably fixed as it was in v3.2.0 and since then we did sdk upgrade.
